sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineTranslator.java

```diff
@@ -725,6 +729,16 @@ private void addOutput(String name, PValue value, Coder<?> valueCoder) {
       outputInfoList.add(outputInfo);
     }
 
+    private void addDisplayData(String name, DisplayData displayData) {
+      List<Map<String, Object>> serializedItems = Lists.newArrayList();
+      ObjectMapper mapper = MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
Does this mutate MAPPER? 
If yes, isn't that a problem?
If no, would it make sense to have a separate static "NON_NULL_MAPPER" so we don't need to create a new mapper for every step? Any performance risk there?